src/main/java/net/minecraftforge/energy/CapabilityEnergy.java

```diff
@@ -27,21 +27,50 @@
 import net.minecraftforge.common.capabilities.*;
 import net.minecraftforge.common.capabilities.Capability.IStorage;
 
+/**
+ * This class provides a static method to register the Energy capability.
+ * It also injects the Energy capability into the CapabilityManager.
+ */
```

Comment:
This can be kept, the rest of the comments are redundant

Reply:
Done!